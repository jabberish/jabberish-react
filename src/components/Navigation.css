.Navigation {
  .Link {
    padding-right: 10px;
  }
}