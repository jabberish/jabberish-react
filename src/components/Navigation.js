import React from 'react';
import { Link } from 'react-router-dom';

import styles from './Navigation.css';
import { Button } from '@material-ui/core';

const Navigation = () => {
  return (
    <nav className={styles.Navigation}>
      <Link className={styles.Link} to='/'><Button color="inherit">Home</Button></Link>
      <Link className={styles.Link} to='/register'><Button color="inherit">Register</Button></Link>
      <Link className={styles.Link} to='/login'><Button color="inherit">Login</Button></Link>
    </nav>
  );
};

export default Navigation;
