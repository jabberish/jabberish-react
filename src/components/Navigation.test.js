import React from 'react';
import { shallow } from 'enzyme';
import Navigation from './Navigation';

describe('Navigation component', () => {
  it('renders Navigation', () => {
    const wrapper = shallow(<Navigation />);
    expect(wrapper).toMatchSnapshot();
  });
});
  
