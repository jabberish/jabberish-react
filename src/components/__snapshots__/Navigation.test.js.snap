// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation component renders Navigation 1`] = `
<nav>
  <Link
    to="/"
  >
    Home
  </Link>
  <Link
    to="/landing"
  >
    Landing
  </Link>
  <Link
    to="/workspace"
  >
    Workspace
  </Link>
  <Link
    to="/register"
  >
    Register
  </Link>
  <Link
    to="/login"
  >
    Login
  </Link>
</nav>
`;
